UnityProject/Assets/Scripts/Core/Input System/CablePlacementVisualisation.cs

```diff
@@ -151,8 +151,9 @@ private void UpdateMe()
 	private void Build()
 	{
 		if (startPoint == endPoint || target == null) return;
+		var Position = cablePlacementVisualisation.transform.position + new Vector3(0.5f, 0.5f, 0);
 
-		Vector2 targetVector = (connectionPointRenderers[endPoint].transform.position - transform.position);
+		Vector2 targetVector = Position.ToLocal(PlayerManager.LocalPlayer.RegisterTile().Matrix); // transform.position ( - transform.position); //TODO? what? is this
```

Comment:
Have you tested placing cables after this change?

Reply:
it works fine